src/pages/cookbooks/[cookbookId]/recipes/[recipeId]/sections/[sectionId].tsx

```diff
@@ -1,11 +1,45 @@
 import { useRouter } from 'next/router';
-import { FunctionComponent } from 'react';
+import { useEffect } from 'react';
+import { useStore } from '../../../../../../store/store';
+import { CookbookSection } from '../../../../../../components/Cookbook/CookbookGuide/CookbookSection/CookbookSection';
+import { SectionModel } from '../../../../../../models/Section';
 
-export const Section: FunctionComponent = () => {
+const init = async (sectionId: string | string[]) => {
+  if (sectionId != null) {
```

Comment:
Would reverse the condition and return early to reduce nesting

Reply:
+1